.github/workflows/docker.yml

```diff
@@ -15,14 +15,21 @@ jobs:
       -
         name: Check out
         uses: actions/checkout@v3
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
       -
         name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
+      -
+        name: Available platforms
+        run: echo ${{ steps.buildx.outputs.platforms }}
       -
         name: Build and export to Docker
         uses: docker/build-push-action@v2
         with:
           context: .
+          platforms: linux/amd64,linux/arm64
```

Comment:
Build fails with:

>  Error: buildx failed with: error: docker exporter does not currently support exporting manifest lists

Reply:
See https://github.com/docker/buildx/issues/59. Looks like this can be fixed by doing the following:

  1. Build without `platforms` and with `load`.
  2. Test.
  3. Set up QEMU.
  4. Build again, this time with `platforms`, without `load`, and with `push`.

In other words, remove the first `platforms` line and move QEMU setup to just before the second build.

Reply:
Done ;-)